src/model/schema.js

```diff
@@ -465,7 +465,7 @@ export class MarkType extends SchemaItem {
 // with node type constructors and another similar object associating
 // mark names with mark type constructors.
 export class SchemaSpec {
-  // :: (?Object<NodeType>, ?Object<MarkType>)
+  // :: (?Object<SubClass<NodeType>>, ?Object<SubClass<MarkType>>)
```

Comment:
Ah, I see what you mean. `Subclass`, as a term, doesn't seem to clarify much -- if `X` stands for an instance of class `X`, then `Subclass<X>` sounds like it stands for an instance of any subclass of `X`. Maybe `Constructor<NodeType>`? Also not terribly elegant. Do you know of any convention (from Flow, TypeScript, JSDoc, whatever) that we can steal?